influunt-app/app/views/main.html

```diff
@@ -138,10 +141,15 @@ <h2 class="font-bold">{{modosOperacoes['MANUAL'] || 0}}</h2>
   <div class="col-lg-4 col-sm-12">
     <div class="ibox float-e-margins white-bg">
       <div class="ibox-content ibox-heading yellow-bg">
-        <h3><strong>{{errosControladores.length || ('main.nenhum' | translate)}}</strong> {{ 'main.controlApresentando' | translate }} <strong>{{ 'main.falha' | translate }}</strong></h3>
+        <h3 data-ng-bind-html="'main.falhasRegistradas' | translate:{ FALHAS: errosControladores.length || 0 }">
+
+          <!-- <strong>{{ errosControladores.length || ('main.nenhum' | translate) }}</strong>
```

Comment:
Isso vai ficar comentado? Se for descomentar, é uma boa alterar o texto para levar em conta a pluralização das palavras (estava mostrando "3 controlador está apresentando falhas").